src/data/python-quiz.ts

```diff
@@ -1,4 +1,24 @@
 const pythonQuiz = [
+  {
+    Question: "In Python, which of the following method returns last value from a list?",
+    Answer: "pop()",
+    Distractor1: "remove()",
+    Distractor2: "last()",
+    Distractor3: "end()",
+    Explanation:
+      "By default, if there is no index specified, the pop() method will remove the last item that is contained in the list. Besides just removing the item, pop() also returns it.",
+    Link: "https://www.freecodecamp.org/news/python-pop-how-to-pop-from-a-list-or-an-array-in-python/#howtousethepopmethodwithnoparameteranamenoparametera",
```

Comment:
We already have a question on the pop method for this Python quiz.
Please remove this duplicate question.